th2_data_services_lwdp/utils/misc.py

```diff
@@ -1,18 +1,47 @@
 from datetime import datetime, timezone
+from th2_data_services.interfaces.utils.converter import ITimestampConverter, TimestampType
+
+
+class DatetimeConverter(ITimestampConverter):
+    @classmethod
+    def parse_timestamp(cls, timestamp: TimestampType) -> (str, str):
+        if isinstance(timestamp, datetime):
+            timestamp = timestamp.replace(tzinfo=timezone.utc).timestamp()
+
+        seconds = int(timestamp)
+        nanoseconds = int((timestamp - seconds) * 1_000_000_000)
+        return tuple(map(str, (seconds, nanoseconds)))
+
+    @classmethod
+    def to_milliseconds(cls, timestamp: TimestampType):
```

Comment:
Move, please, this method to ITimestampConverter

Reply:
![image](https://user-images.githubusercontent.com/25672975/211246234-8b106055-1c84-4674-9f6c-53bc5b391339.png)

moreover, I asked you to do it.

Reply:
Move to copied ProtobufTimestampConverter